ui-frontend/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -227,6 +227,10 @@ module.exports = new (Backbone.Model.extend({
     return metacardTypes.sort((a, b) => {
       const attrToCompareA = (a.alias || a.id).toLowerCase()
       const attrToCompareB = (b.alias || b.id).toLowerCase()
+      if (a.id == 'anyText') return -1
```

Comment:
✏️ Just a suggestion/option, but it might be a bit easier to add new values (or eventually make an admin config) if we do something like:

```
const PRIORITY_ATTRIBUTES = ["anyText", "anyGeo"] // or some better name - outside the function

if (PRIORITY_ATTRIBUTES.includes(a.id)) return -1
if (PRIORITY_ATTRIBUTES.includes(b.id)) return 1
```